frame/ranked-collective/src/lib.rs

```diff
@@ -683,5 +662,49 @@ pub mod pallet {
 			}
 			Ok(())
 		}
+
+		/// Decrement the rank of an existing member by one. If the member is already at rank zero,
+		/// then it is removed entirely.
+		///
+		/// A `maybe_max_rank` may be provided to check that the member does not get demoted beyond
+		/// a certain rank. Is `None` is provided, then the rank will be decremented without checks.
```

Comment:
I find this comment also confusing. Without reading the code, I wouldn't really know what this is for.